addons/wowl/static/src/legacy/legacy.ts

```diff
@@ -262,6 +266,7 @@ odoo.define("wowl.legacyViews", async function (require: any) {
       static icon = LegacyView.prototype.icon;
       static multiRecord = LegacyView.prototype.multi_record;
       static type = LegacyView.prototype.viewType;
+      static isLegacy = true as true;
```

Comment:
lol; true as true ?

Reply:
It is as true as true it is.
https://www.youtube.com/watch?v=AR8D2yqgQ1U